common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
```

Comment:
If we do end up introducing this file, please put it in the spec folder so it is clear that it is only for tests.